package.json

```diff
@@ -69,6 +69,7 @@
     "clean": "rm -rf node_modules package-lock.json",
     "contributors": "(git-authors-cli && finepack && git add package.json && git commit -m 'build: contributors' --no-verify) || true",
     "coverage": "jest --coverage --coverageReporters=text-lcov | coveralls",
+    "coverage:dev": "jest --coverage",
```

Comment:
This is not needed, our `npm test` is already this command 😉

Reply:
Ah ha, I overlooked it :)

Reply:
Fixed 2077b55